storage/rocksdb/ha_rocksdb.cc

```diff
@@ -17088,13 +17093,13 @@ bool ha_rocksdb::is_dd_update() const {
   }
 
 #define DEF_STATUS_VAR(name) \
-  { "rocksdb_" #name, (char *)&SHOW_FNAME(name), SHOW_FUNC, SHOW_SCOPE_GLOBAL }
+  {"rocksdb_" #name, (char *)&SHOW_FNAME(name), SHOW_FUNC, SHOW_SCOPE_GLOBAL}
```

Comment:
our internal linter complain this change.. it recommend to add space ..:)

Reply:
Which clang-format version does your linter use?

Reply:
all of these lines.

Reply:
> Which clang-format version does your linter use?

clang-format --version
clang-format version 18.1.3

Reply:
@laurynas-biveinis , are you using clang-format 19? or 

If I run your change with clang-format 18, it will show following output
--- a/storage/rocksdb/ha_rocksdb.cc
+++ b/storage/rocksdb/ha_rocksdb.cc
@@ -17093,13 +17093,13 @@ bool ha_rocksdb::is_dd_update() const {
   }

 #define DEF_STATUS_VAR(name) \
-  {"rocksdb_" #name, (char *)&SHOW_FNAME(name), SHOW_FUNC, SHOW_SCOPE_GLOBAL}
+  { "rocksdb_" #name, (char *)&SHOW_FNAME(name), SHOW_FUNC, SHOW_SCOPE_GLOBAL }

 #define DEF_STATUS_VAR_PTR(name, ptr, option) \
-  {"rocksdb_" name, (char *)ptr, option, SHOW_SCOPE_GLOBAL}
+  { "rocksdb_" name, (char *)ptr, option, SHOW_SCOPE_GLOBAL }

 #define DEF_STATUS_VAR_FUNC(name, ptr, option) \
-  {name, reinterpret_cast<char *>(ptr), option, SHOW_SCOPE_GLOBAL}
+  { name, reinterpret_cast<char *>(ptr), option, SHOW_SCOPE_GLOBAL }

Reply:
@luqun , rebased and reformatted, ready for review

I am using clangd 19, which effectively should be clang-format 19. Interestingly if I run this file through clang-format 18.1.5 after it is already formatted with 19, the diff is empty. I have re-added the spaces manually.